src/bytes.rs

```diff
@@ -507,6 +507,28 @@ impl Bytes {
         self.truncate(0);
     }
 
+    /// Try to convert self into `BytesMut`.
+    ///
+    /// If `self` is unique, this will succeed and return a `BytesMut` with
+    /// the contents of `self` without copying. If `self` is not unique, this
+    /// will fail and return self.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use bytes::{Bytes, BytesMut};
+    ///
+    /// let bytes = Bytes::from(b"hello".to_vec());
+    /// assert_eq!(bytes.try_into_mut(), Ok(BytesMut::from(&b"hello"[..])));
+    /// ```
+    pub fn try_into_mut(self) -> std::result::Result<BytesMut, Bytes> {
```

Comment:
This use of `std` fails in builds without the standard library.
```suggestion
    pub fn try_into_mut(self) -> Result<BytesMut, Bytes> {
```